src/OpenTelemetry/BatchExportProcessor.cs

```diff
@@ -69,6 +71,7 @@ public abstract class BatchExportProcessor<T> : BaseExportProcessor<T>
             Guard.ThrowIfOutOfRange(exporterTimeoutMilliseconds, min: 0);
 
             this.circularBuffer = new CircularBuffer<T>(maxQueueSize);
+            this.circularBufferDroppedCount = Sdk.SdkInternalStateMeter.CreateObservableCounter($"{typeof(T).Name}.BatchExportProcessorDroppedCount", () => this.DroppedCount);
```

Comment:
What's the lifecycle management story? (would the static Meter hold reference to all instances of batch processor until the proc terminates?)